userguide/docs/java.mdx

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Checking for and Installing Java
+
+This section provides instructions on how to check your computer for an existing Java installation.
+
+If you are sure that you do not already have Java installed, you can skip to the [Installing Java] section of this article below.
+
+## Checking for Existing Java Installation
+
+To find out if you already have Java, you will need to locate and run the terminal application that is provided by your OS.
+
+### Location Terminal Application
```

Comment:
It's probably best to specify that these are shells that come pre-installed with the OS. If the user decides to install their own shell, then it is up to them to know what the application is called.

Reply:
Good point. I'll clean that up :+1: